model/job-management/src/main/proto/beam_artifact_api.proto

```diff
@@ -31,8 +31,77 @@ option java_outer_classname = "ArtifactApi";
 
 import "beam_runner_api.proto";
 
-// A service to stage artifacts for use in a Job.
+// A service to retrieve artifacts for use in a Job.
+service ArtifactRetrievalService {
+  // Resolves the given artifact reference into one or more simpler artifact
+  // references (e.g. a Maven dependency into a (transitive) set of jars.
+  // If no further simplification is possible, returns the original artifacts,
```

Comment:
What does simpler mean?, should the request contain a supported set of urns for types and roles?

Reply:
Clarified in the docs.